integration/rpctest/rpc_harness.go

```diff
@@ -482,8 +483,8 @@ func generateListeningAddresses() (string, string) {
 	localhost := "127.0.0.1"
 
 	portString := func(minPort, maxPort int) string {
-		port := minPort + numTestInstances + ((20 * processID) %
-			(maxPort - minPort))
+		rand.Seed(time.Now().UnixNano())
+		port := minPort + numTestInstances + rand.Intn(maxPort-minPort)
```

Comment:
Why not just `minPort + rand.Intn(maxPort-minPort)`?

Reply:
You'll need to also update the comments in the lines 479 through 481, as the ports are not incremented anymore, but rather random.

It also seems `processID` doesn't serve any purpose anymore, so it can be removed too.

Reply:
👍